doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -261,6 +261,18 @@ Modem libraries
   * Removed ``MODEM_INFO_NETWORK_MODE_MAX_SIZE``.
   * Removed ``CONFIG_MODEM_INFO_ADD_BOARD``.
 
+* :ref:`nrf_modem_lib_readme`:
+
+  * Updated:
+
+    * The :c:func:`getaddrinfo` function to return ``EAFNOSUPPORT`` instead of ``EPROTONOSUPPORT`` when socket family is not supported.
+    * The :c:func:`bind` function to return ``EAFNOSUPPORT`` instead of ``ENOTSUP`` when socket family is not supported.
+    * The :c:func:`sendto` function to return ``EAFNOSUPPORT`` instead of ``ENOTSUP`` when socket family is not supported.
+    * The :c:func:`connect` function to not override the error codes set by the modem library when called with raw parameters (non-IP).
```

Comment:
```suggestion
    * The :c:func:`getaddrinfo` function to return ``EAFNOSUPPORT`` instead of ``EPROTONOSUPPORT`` when socket family is not supported.
    * The :c:func:`bind` function to return ``EAFNOSUPPORT`` instead of ``ENOTSUP`` when socket family is not supported.
    * The :c:func:`sendto` function to return ``EAFNOSUPPORT`` instead of ``ENOTSUP`` when socket family is not supported.
    * The :c:func:`connect` function to not override the error codes set by the Modem library when called with raw parameters (non-IP).
```